### ISSUE TITLE
forgetti 0.6.0 pre-stable issues

### ISSUE THREAD

**Issue by SukkaW** [lxsmnsyc/forgetti#25]
https://github.com/lxsmnsyc/forgetti/pull/23

----

<details>
<summary>Fixed</summary>

**Version**: 0.6.0-alpha.0

**Input**

```jsx
import { StrictMode } from 'react';
import { RouterProvider } from 'react-router-dom';
import { HelmetProvider } from 'react-helmet-async';

import { router } from './router';

export default function App() {
  return (
    <StrictMode>
      <HelmetProvider>
        <RouterProvider router={router} />
      </HelmetProvider>
    </StrictMode>
  );
}
```

**Output**

```jsx
import { useRef as _useRef } from "react";
import { $$cache as _$$cache } from "forgetti/runtime";
import { memo as _memo } from "react";
import { $$memo as _$$memo } from "forgetti/runtime";
const _Memo = _$$memo(_memo, _values => {
  const _Component = StrictMode,
    _Component2 = HelmetProvider,
    _Component3 = RouterProvider;
  return <_Component>
      <_Component2>
        <_Component3 router={_values[0]} />
      </HelmetProvider>
    </StrictMode>;
});
import { StrictMode } from 'react';
import { RouterProvider } from 'react-router-dom';
import { HelmetProvider } from 'react-helmet-async';
import { router } from './router';

/** @forgetti skip */
export default function App() {
  let _cache = _$$cache(_useRef, 1);
  return 0 in _cache ? _cache[0] : _cache[0] = /*@forgetti skip*/<_Memo v={[router]} />;
}
```

The closing element of `<_Component>` is not `</_Component>` but remains `</StrictMode>`.
</details>

**Comment by lxsmnsyc** [comment]
@SukkaW Just released `0.6.0-alpha.2`, should be fixed now (as well as for local component usage and member components)

**Comment by SukkaW** [comment]
> @SukkaW Just released `0.6.0-alpha.2`, should be fixed now (as well as for local component usage and member components)

Confirmed the issue has been fixed. Awesome！
